src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-cv2.py

```diff
@@ -31,3 +31,5 @@
     # Include any DLLs from site-packages/cv2 (opencv_videoio_ffmpeg*.dll
     # can be found there in the PyPI version)
     binaries += collect_dynamic_libs('cv2')
+    # Need to include extra config files and modules
+    extra_data = collect_data_files('cv2', include_py_files=True, includes=['**\*.py'])
```

Comment:
Surely it should be `includes=['**/*.py']`?

Reply:
> Surely it should be `includes=['**/*.py']`?

Uh, right, my bad.

And in addition, it should also be `datas` and not `extra_data`. And it should be outside the if block to be OS-agnostic...

Reply:
Thanks for the hint!